qcodes/station.py

```diff
@@ -52,6 +53,9 @@ def get_config_use_monitor() -> Optional[str]:
     return qcodes.config["station"]["use_monitor"]
 
 
+ChannelOrInstrumentBase = Union[InstrumentBase, ChannelList]
```

Comment:
shouldn't this also be a channel? (not only channel lists) some drivers have "submodules" with parameters implemented via subclassing InstrumentChannel.
```suggestion
ChannelOrInstrumentBase = Union[InstrumentBase, ChannelList, InstrumentChannel]
```

Reply:
It is :-)
```python
class InstrumentChannel(InstrumentBase):
```

Reply:
oh that's super good! stupid me :)